### ISSUE TITLE
master is currently broken

### ISSUE THREAD

**Issue by Stanzilla** [KELiON/cerebro#423]
Errors with 
![cerebro_2017-12-02_20-11-23](https://user-images.githubusercontent.com/75278/33518887-1d1076c6-d79d-11e7-8b2c-a74a17f52337.png)

on start

**Comment by lawgsy** [comment]
I can compile it fine, can you give us more information about your environment? Did you install using a recent version of yarn?

**Comment by Stanzilla** [comment]
that was with yarn 1.3.2 and `yarn && yarn run release`

**Comment by lawgsy** [comment]
Yes I am. I'm surprised you got that far into running `yarn run release` (I am getting an error about being unable to compile for macOS from a Windows computer, because the release script of `package.json` contains the flag `-wml`, which includes m for mac).

Can you try the following from your `cerebro\` directory?
1.  `yarn && cd app && yarn` to install all dependencies
2. `yarn run package` to build and package (required for the first time to generate CSS in the right directories when running without packaging later)
3. `yarn run start` to run in production or `yarn run dev` to run in development setting

After step 2 succeeds, you should also find a binary in `cerebro\release\win-unpacked`

**Comment by Stanzilla** [comment]
I think I already found the problem, Duplicate of https://github.com/KELiON/cerebro/issues/426 I guess

**Comment by KELiON** [comment]
@Stanzilla nope, #426 is another issue:) This one was fixed [here](https://github.com/KELiON/cerebro/commit/b379a5b90ba63df99155210ef043d69ae785e6a6)

**Comment by Stanzilla** [comment]
oh durr! thanks!

**Comment by lawgsy** [comment]
Cannot reproduce new issue. It appears the current binaries currently produce the error you mentioned at the start of the issue though.

**Comment by KELiON** [comment]
@Stanzilla what version of node did you use, when you had second issue? Probably it is somehow related to #426

**Comment by ldexterldesign** [comment]
+1

<img width="1440" alt="screen shot 2018-06-02 at 16 53 09" src="https://user-images.githubusercontent.com/1715809/40877713-b4c2afd0-6685-11e8-98cc-c36eb9b2c315.png">

Let me know if you have any issues - happy to help

Regards
